docker_manager.py

```
@@ -0,0 +1,304 @@
+import docker
+import logging
+import asyncio
```

Ruff (0.12.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)